invenio_app_ils/ui/src/invenio_app_ils/common/api/loans/loan.js

```diff
@@ -203,6 +203,16 @@ const queryBuilder = () => {
   return new QueryBuilder();
 };
 
+const fetchFrom = link => {
```

Comment:
In my opinion, we should not add a new method here, but instead use `list` and pagination. To be re-discussed.